app/controllers/api/groups_controller.rb

```diff
@@ -1,6 +1,12 @@
 class API::GroupsController < API::RestfulController
   load_and_authorize_resource only: :show, find_by: :key
   load_resource only: [:upload_photo, :use_gift_subscription], find_by: :key
+  skip_before_action :authenticate_user!, only: :index
```

Comment:
Yea.. At some point Hannah and I did that. I think Hannah may have moved it
back trying to fix broken specs. Note to all that I think BaseController
should be renamed to LoggedInController

On Mon, 18 Apr 2016 at 12:06 James Kiesel <notifications@github.com> wrote:

> In app/controllers/api/groups_controller.rb
> <https://github.com/loomio/loomio/pull/3194#discussion_r59996005>:
>
> > @@ -1,6 +1,12 @@
> >  class API::GroupsController < API::RestfulController
> >    load_and_authorize_resource only: :show, find_by: :key
> >    load_resource only: [:upload_photo, :use_gift_subscription], find_by: :key
> > +  skip_before_action :authenticate_user!, only: :index
>
> I was able to fix by moving include AngularHelper from BaseController to
> ApplicationController, then changing the route from base#boot_angular_ui
> to application#boot_angular_ui
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/loomio/loomio/pull/3194/files/c9b754ca5d38df38394934bf06019f4654a794da#r59996005>
>